pyvrp/cpp/DurationSegment.h

```diff
@@ -178,8 +206,9 @@ Duration DurationSegment::duration() const { return duration_; }
 Duration DurationSegment::timeWarp(Duration const maxDuration) const
 {
     // clang-format off
-    return timeWarp_
-         + std::max<Duration>(releaseTime_ - twLate_, 0)
+    Duration const latestStart = std::max<Duration>(latestFinish_ - duration_, earliestStart_);
```

Comment:
If we inline `DurationSegment::latestStart()`, we can call the method here. That saves a bit on having to understand the computation in two places.

Reply:
Good idea, thanks :)